src/main/java/com/minecolonies/achievements/MineColoniesAchievement.java

```diff
@@ -19,44 +19,41 @@
      * Create a new Achievement.
      *
      * @param id       the unique id this achievement should have
-     * @param name     the visible name this achievement should have
      * @param offsetX  X position placement on the board
      * @param offsetY  Y position placement on the board
      * @param itemIcon The icon to show
      * @param parent   the parent achievement
      */
-    public MineColoniesAchievement(final String id, final String name, final int offsetX, final int offsetY, final Item itemIcon, final Achievement parent)
```

Comment:
We should prefix all of our achievement IDs with "minecolonies." So that we are sure we won't have conflicts with other mods. "supply" and "wandofbuilding" are pretty generic.